tests/test_order.py

```diff
@@ -26,26 +23,19 @@ def test_repr(auth_mock):
         "createdAt": "2023-01-01T12:00:00Z",
         "updatedAt": "2023-01-01T12:30:00Z",
     }
-    order = Order(auth_mock, order_id="your_order_id", order_info=order_info)
+    order_placed = order.Order(auth_mock, order_id="your_order_id", order_info=order_info)
 
     expected_repr = (
         "Order(order_id: your_order_id, status: PLACED,"
         "createdAt: 2023-01-01T12:00:00Z, updatedAt: 2023-01-01T12:30:00Z)"
     )
-    assert repr(order) == expected_repr
-
-
-@pytest.mark.live
-def test_order_info_live(order_live):
-    assert order_live.info
-    assert order_live.info["id"] == os.getenv("TEST_UP42_ORDER_ID")
-    assert order_live.info["dataProductId"] == "4f1b2f62-98df-4c74-81f4-5dce45deee99"
+    assert repr(order_placed) == expected_repr
 
 
 # pylint: disable=unused-argument
```

Comment:
Not needed

Reply:
@javidq but this is needed in an another test.
Please see the latest changes